deployments/helm/crds/skupper_access_grant_crd.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Ideally the CRDs would be defined in one place or copied by the build. Otherwise it is easy to make changes to one copy and not the other.